src/main/resources/com/owlcyberdefense/ntp/xsd/ntp.dfdl.xsd

```diff
@@ -1,29 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <schema
-  xmlns:xs="http://www.w3.org/2001/XMLSchema"
-  xmlns="http://www.w3.org/2001/XMLSchema"
-  xmlns:fn="http://www.w3.org/2005/xpath-functions"
-  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
-  xmlns:ntp="urn:ntp"
-  targetNamespace="urn:ntp"
-  elementFormDefault="unqualified">
+	xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
At some point, indentation, (coding standard) to be applied.

Usually 2-space indent in DFDL schemas.